Formula/operator-sdk.rb

```diff
@@ -13,31 +13,36 @@ class OperatorSdk < Formula
   end
 
   bottle do
```

Comment:
This block should not be modified

Reply:
Should be fixed as of commit https://github.com/Homebrew/homebrew-core/pull/100353/commits/9d2700a770552f456b1448b1eadc6ba43db10769